Makefile

```diff
@@ -26,6 +27,11 @@ push: image
 	docker push $(REPO):$(HASH)
 	docker push $(REPO):$(DATE)-$(VERS)-$(HASH)
 
+release:
+	# push the local 'latest' tag as the new public latest version
+	# (run by CI only for releases)
+	docker push $(REPO):latest
```

Comment:
Don't we also want to push `$(VERS)` tag?

Reply:
Yes, and that's a good question; I should clarify: there is a make target `make push` for pushing VERS, HASH, and teh verbose DATE-VERS-HASH, which can be invoked at any time, even from an unreleased, unversioned commit such as from a feature branch.  In this case VERS will just be `tip`.  However, when the GitHub action (`.github/workflows/releases.yaml`) is run, it will `make push` but, since it only matches versioned commits, it will _also_ run this release target, which essentially tags the already-created `:$(VERS)` image as `:latest`.